generator/sig_readme.tmpl

```diff
@@ -25,9 +25,9 @@ Meeting recordings can be found [here]({{.MeetingRecordingsURL}}).
 {{- if .Label }}
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2F{{.Label}})
 {{- end }}
-{{if .Contact.FullGitHubTeams}}
-## GitHub Teams
-{{range .Contact.GithubTeamNames -}}
-* [@{{.}}](https://github.com/kubernetes/teams/{{.}})
+{{if .Contact.GithubTeams}}
+### GitHub Teams
+{{range .Contact.GithubTeams -}}
+* **[@kubernetes/{{.}}](https://github.com/orgs/kubernetes/teams/{{.}})**
```

Comment:
nit: maybe not italics here?

Reply:
Two stars is bold not italics. What it looks like on my branch: https://github.com/cblecker/community/blob/4bbdc7ef49578bb4b4fdd6eead9cbdc69606523d/sig-api-machinery/README.md